[2024-03-22T20:00:10.876+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: milestone3.clean_data scheduled__2024-03-22T19:30:00+00:00 [queued]>
[2024-03-22T20:00:10.888+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: milestone3.clean_data scheduled__2024-03-22T19:30:00+00:00 [queued]>
[2024-03-22T20:00:10.888+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2024-03-22T20:00:10.888+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 2
[2024-03-22T20:00:10.889+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2024-03-22T20:00:10.903+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): clean_data> on 2024-03-22 19:30:00+00:00
[2024-03-22T20:00:10.932+0000] {standard_task_runner.py:52} INFO - Started process 3043 to run task
[2024-03-22T20:00:10.940+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'milestone3', 'clean_data', 'scheduled__2024-03-22T19:30:00+00:00', '--job-id', '324', '--raw', '--subdir', 'DAGS_FOLDER/P2M3_elia_DAG.py', '--cfg-path', '/tmp/tmp27xq347r', '--error-file', '/tmp/tmp3pz3q8tv']
[2024-03-22T20:00:10.942+0000] {standard_task_runner.py:80} INFO - Job 324: Subtask clean_data
[2024-03-22T20:00:11.035+0000] {task_command.py:371} INFO - Running <TaskInstance: milestone3.clean_data scheduled__2024-03-22T19:30:00+00:00 [running]> on host b79cb602ca97
[2024-03-22T20:00:11.131+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=elia
AIRFLOW_CTX_DAG_ID=milestone3
AIRFLOW_CTX_TASK_ID=clean_data
AIRFLOW_CTX_EXECUTION_DATE=2024-03-22T19:30:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-03-22T19:30:00+00:00
[2024-03-22T20:00:11.698+0000] {python.py:173} INFO - Done. Returned value was: None
[2024-03-22T20:00:11.727+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=milestone3, task_id=clean_data, execution_date=20240322T193000, start_date=20240322T200010, end_date=20240322T200011
[2024-03-22T20:00:11.781+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-03-22T20:00:11.848+0000] {local_task_job.py:279} INFO - 1 downstream tasks scheduled from follow-on schedule check
